spec/presenters/two_factor_auth_code/phone_delivery_presenter_spec.rb

```diff
@@ -18,12 +18,41 @@
       reauthn: false,
     }
   end
+  let(:data_for_locale) do
```

Comment:
Looks like a lot of this is borrowed from data. Also, it doesn't look like this is actually used in a spec. What are your thoughts on adding a method to this spec, say `present_with_locale(locale)`, that returns are presenter for the given locale?

That way you wouldn't need this hash, and in that method you could merge the necessary values into the `data` hash.

Reply:
Yes. I tried to follow same format.  Your idea sounds cleaner.  Only negative is it would be slower  since in this version you init data once and then reference it vs assembling it on each call.  However there are only a few calls so speed isn't really an issue.   I think the un-dry-ness comes from the fact that I dup'd the data (since I wasn't too certain if both would diverge and how the other was being used).  I could refer to the original hash when assembling the new one and that should make it a little dry-er. Your thoughts?

Reply:
I guess my "refer to original hash" would be your present_with_locale(locale)

Reply:
Yeah, something like this:

```ruby
def presenter_with_locale(locale)
  data_with_locale = data.merge(locale: locale)
  Presenter.new(...)
end
```

Reply:
ok I'll make the change... Thanks!